### ISSUE TITLE
Log referer

### ISSUE THREAD

**Issue by fra967** [dadi/web#189]
Please add a configuration option so that http referer header can be added to the access logs

**Comment by jimlambie** [comment]
@fra967 thanks for this. It is actually already logged via the @dadi/logger module. Below is the result of a test run earlier today:

### Request headers received by logger
```
[2017-07-06 09:21:23.391] [LOG]   { host: '54.194.34.92:8000',
  connection: 'keep-alive',
  'upgrade-insecure-requests': '1',
  accept: 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8',
  'user-agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/603.1.30 (KHTML, like Gecko) Version/10.1 Safari/603.1.30',
  referer: 'http://52.209.115.88/jim/',
  'accept-language': 'en-us',
  'accept-encoding': 'gzip, deflate' }
```

### Access log

`$ tail -f log/web.access.log`

```
{"name":"access","hostname":"james.net","pid":17620,"level":30,"msg":"178.62.21.169 - 2017-07-06T09:21:16+00:00 GET / HTTP/1.1 200 5430 Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/603.1.30 (KHTML, like Gecko) Version/10.1 Safari/603.1.30","time":"2017-07-06T09:21:16.916Z","v":0}
{"name":"access","hostname":"james.net","pid":17620,"level":30,"msg":"178.62.21.169 - 2017-07-06T09:21:23+00:00 GET / HTTP/1.1 200 5430 http://52.209.115.88/jim/ Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/603.1.30 (KHTML, like Gecko) Version/10.1 Safari/603.1.30","time":"2017-07-06T09:21:23.393Z","v":0}
```

**Comment by fra967** [comment]
ok thank you @jimlambie , referer is missing in Web 1.10.5 - Logger 1.2.0

Since which version is supported?

**Comment by jimlambie** [comment]
@fra967 available in logger since at least 1.1.0, and should be in [Web 1.10.5](https://github.com/dadi/web/blob/v1.10.5/dadi/lib/index.js#L118)

The referer is added to the request headers by Node's http server - if it's available in the raw request (HTTP_REFERER) then it should be in the object Web/Logger receive.

**Comment by fra967** [comment]
yes I can confirm it's logged when passed directly, something else must be stripping it
